developing/command-line-tools/README.md

```diff
@@ -88,7 +88,7 @@ You may also be interested in these other pages from the Truss Engineering Playb
 
 ## Document format conversion
 
-- [dos2unix](https://waterlan.home.xs4all.nl/dos2unix.html) Efficient conversion of line terminators between DOS <-> Unix format
+- [dos2unix](https://waterlan.home.xs4all.nl/dos2unix.html) Efficient conversion of line terminators between DOS \<-> Unix format
```

Comment:
Why is this backslash here?

Reply:
If it bothers you, we could change this to `⬌` or `⇆` or such. Or just "and".